src/LineBot/RichMenu/RichMenuIdResponse.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright 2017-2018 Dirk Lemstra (https://github.com/dlemstra/line-bot-sdk-dotnet)
+//
+// Dirk Lemstra licenses this file to you under the Apache License,
+// version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at:
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+using Newtonsoft.Json;
+
+namespace Line
+{
+    /// <summary>
+    /// Rich menu id response object. This object is returned when you get a rich menu or get a list of rich menus.
+    /// </summary>
+    internal sealed class RichMenuIdResponse
```

Comment:
Please remove this file. We don't need it anymore?

Reply:
This class is used to get the richMenuIdId from json response like below.
```
"{""richMenuId"": ""richmenu-801b2cd26b2f13587329ed501d279d27""}"
```
Is there any better way to just get the Id?

Reply:
I thought my comment was for the interface. We do still need this class.